chanzuckerberg/single-cell-data-portal
# fix: 6372 GE image download

## Reason for Change

- #6372
- If the reason for this PR's code changes are not clear in the issue, state value/impact

## Changes

1. Not a direct fix to eliminate duplicate font file calls, but upping network guard to 500 per 5s should be safe anyway

## Testing steps

1. To go GeneExpression in PR rdev, you should be able to download images again
2. Observe that there are ~84 font file calls

https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/7466d753-c2d4-4f2a-9bc2-a6b5e895f820

## Checklist 🛎️


## Notes for Reviewer
